tools/repoctl/src/repoctl/affected.py

```
@@ -0,0 +1,28 @@
+             "native/edge-runtime": "T14", "native/firmware-imu": "T6"}
+
+def assets_for(paths):
+    out = set()
+    for p in paths:
+        parts = Path(p).parts
+        if len(parts) >= 3 and parts[0] == "packages" and (a := PKG_ASSET.get(f"{parts[1]}/{parts[2]}")):
+            out.add(a)
+    return sorted(out, key=lambda t: int(t[1:]))
+
+def run(args) -> int:
+    r = subprocess.run(["git", "-C", args.root, "diff", "--name-only", args.base],
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)